proof/curve_operations.saw

```diff
@@ -608,17 +598,18 @@ POINTonE1_base_thms <- for
          if c then POINTonE1_rep p1 else POINTonE1_rep p2 }}
  , {{ \c p1 p2 -> POINTonE1_abs (if c then p1 else p2) ==
          if c then POINTonE1_abs p1 else POINTonE1_abs p2 }}
-
+//
  , {{ \c p1 p2 -> point_neg Fp (if c then p1 else p2) ==
                   if c then point_neg Fp p1 else point_neg Fp p2 }}
-
- , {{ \ (x:POINTonE1_t) -> (x.0, x.1, x.2) == x }}
- , {{ \ (x:POINTonE1_affine_t) -> (x.0, x.1) == x }}
+//
+ , {{ \(x:POINTonE1_t) -> (x.0, x.1, x.2) == x }}
```

Comment:
I tend to put a space between a backslash and open paren due to a shortcoming the saw-script.el for emacs shipped with saw, which does not parse correctly without it.  It would be better to fix the emacs-lisp code, I suppose, but if there's no strong reason for removing the space, why not leave it in for now?